src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test.Common/LanguageServerTestBase.cs

```diff
@@ -35,17 +35,19 @@ private class SingleThreadedForegroundDispatcher : ForegroundDispatcher
         {
             public SingleThreadedForegroundDispatcher()
             {
-                ForegroundScheduler = SynchronizationContext.Current == null ? new ThrowingTaskScheduler() : TaskScheduler.FromCurrentSynchronizationContext();
+                SpecializedForegroundScheduler = SynchronizationContext.Current == null ? new ThrowingTaskScheduler() : TaskScheduler.FromCurrentSynchronizationContext();
```

Comment:
All files including and below this file can be ignored, as they're all test files and I don't have them set up to pass yet 😃